themes/phenomic-theme-base/scripts/phenomic.browser.js

```diff
@@ -1,26 +1,32 @@
+// Hot loading HRM Patch
+import "react-hot-loader/patch"
```

Comment:
Can UglifyJS remove this in production mode? Why don't you add this to devEntries in dev-server. We shouldn't require any changes in the boilerplate from user.